src/pts/pyspark/timeseries_utils/__init__.py

```diff
@@ -0,0 +1,6 @@
+"""Modules to calculate timeseries."""
+
+from .association import Association
+from .evidence import Evidence
+
+__all__ = ['Association', 'Evidence']
```

Comment:
```suggestion
```

We don't need to limit exports, it adds some maintenance burden because you'll have to remember to keep adding stuff here.

Also, we use absolute imports, but we can remove these from here.

Reply:
You can still leave the re-exports:

```python
from pts.pyspark.timeseries_utils.association import Association
from pts.pyspark.timeseries_utils.evidence import Evidence
```

And just remove the change to `__all__`. That way you get the benefit of shorter imports without having to always add new methods to the list.